### PR TITLE
CEDS-1771 Add tariff link for two new pages

### PR DESCRIPTION
**Pull request by apodi**
* Country of Routing Codes
 * Total Package Quantity

### CODE REVIEW

File: app/views/declaration/destinationCountries/country_of_routing.scala.html [hunk 1]
```diff
@@ -26,6 +26,8 @@
 
 @(mode: Mode, form: Form[String], page: CountryPage)(implicit request: JourneyRequest[AnyContent], messages: Messages)
 
+@tariffBody = { @Html(messages("declaration.routingQuestion.help.bodyText", components.details_content_link())) }
```
Comment (bambuchaAdm) [inline app/views/declaration/destinationCountries/country_of_routing.scala.html]: would you like to inline this varaible ?

File: test/views/declaration/TotalPackageQuantityViewSpec.scala [hunk 2]
```diff
@@ -70,6 +70,11 @@ class TotalPackageQuantityViewSpec extends UnitViewSpec with ExportsTestData wit
           saveButton.text() must be("site.save_and_continue")
         }
 
+        "display Tariff section text" in {
+          val tariffText = view.getElementsByClass("govuk-details__summary-text").first().text()
+          tariffText.text() must be("site.details.summary_text_this")
```
Comment (bambuchaAdm) [inline test/views/declaration/TotalPackageQuantityViewSpec.scala]: please add test for having tanslation key - first test case in suite

File: test/views/declaration/destinationCountries/RoutingCountryQuestionViewSpec.scala [hunk 3]
```diff
@@ -62,6 +61,11 @@ class RoutingCountryQuestionViewSpec extends UnitViewSpec with Stubs with Export
       view.getElementById("No-label").text() mustBe messages("site.no")
     }
 
+    "display Tariff section text" in {
+      val tariffText = view.getElementsByClass("govuk-details__summary-text").first().text()
+      tariffText.text() must be("site.details.summary_text_this")
```
Comment (bambuchaAdm) [inline test/views/declaration/destinationCountries/RoutingCountryQuestionViewSpec.scala]: please add test for translation key

File: app/views/declaration/destinationCountries/country_of_routing.scala.html (hunk 1 above)
Comment (apodi) [reply app/views/declaration/destinationCountries/country_of_routing.scala.html]: done

File: test/views/declaration/TotalPackageQuantityViewSpec.scala (hunk 2 above)
Comment (apodi) [reply test/views/declaration/TotalPackageQuantityViewSpec.scala]: done

File: test/views/declaration/destinationCountries/RoutingCountryQuestionViewSpec.scala (hunk 3 above)
Comment (apodi) [reply test/views/declaration/destinationCountries/RoutingCountryQuestionViewSpec.scala]: done

**Pull request merged**